bitcoin/src/blockdata/locktime/relative.rs

```diff
@@ -124,12 +124,15 @@ impl LockTime {
     /// ```
     #[inline]
     #[cfg_attr(all(test, mutate), mutate)]
-    pub fn is_satisfied_by_height(&self, h: Height) -> Result<bool, Error> {
+    pub fn is_satisfied_by_height(&self, h: Height) -> Result<bool, IncompatibleTimeError> {
```

Comment:
In `abosolute` we have `ConversionError` and `OperationErro`, we should make them consistent within reason.

Reply:
I'll have a look, thanks.

Reply:
Man I found a couple of problems with the new `absolute` error stuff, this error code is hard, huh.

I think the errors are sane in this file. The string parsing is infallible so we do not need the `Conversion:Error` complication.

Then names could be changed if desired, but the code itself is correct I belive.